setup/flex.rst

```diff
@@ -69,27 +74,29 @@ other files and directories. For example, this is the ``manifest.json`` for Swif
     }
 
 The ``aliases`` option allows Flex to install packages using short and easy to
-remember names (``composer require mailer`` vs ``composer require symfony/swiftmailer-bundle``).
-The ``bundles`` option tells Flex in which environments should this bundle be
-enabled automatically (``all`` in this case). The ``env`` option makes Flex to
-add new environment variables to the application. Finally, the ``copy-from-recipe``
-option allows the recipe to copy files and directories into your application.
-
-The instructions defined in this ``manifest.json`` file are also used by Symfony
-Flex when uninstalling dependencies (e.g. ``composer remove mailer``) to undo
-all changes. This means that Flex can remove the SwiftmailerBundle from the
-application, delete the ``MAILER_URL`` environment variable and any other file
-and directory created by this recipe.
+remember names (``composer require mailer`` vs ``composer require
+symfony/swiftmailer-bundle``). The ``bundles`` option tells Flex in which
+environments should this bundle be enabled automatically (``all`` in this
```

Comment:
`environments should this bundle be enabled automatically` should be `environments this bundle should be enabled automatically`